app/helpers/application_helper.rb

```diff
@@ -15,4 +15,12 @@ def like_or_dislike_btn(post)
       link_to('Like!', post_likes_path(post_id: post.id), method: :post)
     end
   end
+
+  # def accept_or_deny_btn(friendship)
+  #   friend_request = Friendship.find_by( user: current_user , friendship: friendship)
+  #   if friend_request
+  #     link_to( 'Accept', user_friendship_path(@user, i.id))
+  #     link_to( 'Deny', user_friendships_path(@user))
+  #   end
+  # end
```

Comment:
It’s better to delete **commented piece of codes**. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))
See this [post](https://stackoverflow.com/questions/13130859/inline-comments-in-ruby/13130900#:~:text=No%2C%20Ruby%20does%20not%20have,comment%20out%20the%20one%20row), about why comments are discouraged.